PrimeNumbers/PrimeNumber_join.cpp

```diff
@@ -98,7 +101,7 @@ const double _1M = pow(10, 6);
 const double _1K = pow(10, 3);
 const char* formatNumber(double input)
```

Comment:
so I think I actually meant to change this arg from double to uint64_t but it got erased when I sync-ed to the latest when submitting that PR. do we actualy need this to be double at all? if not we can just change it to uint64_t

Reply:
fixed